code/src/components/Button.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import "./Button.css";
```

Comment:
I like that you have a css file for each component and that you imported it in each component.
I have done the same, but I imported everything in the [index.js](https://github.com/rawisou/project-survey/blob/master/code/src/index.js) file! 

Looking at your code, this makes much more sense!

I asked Maksy just to be sure there're no big advantages or disadvantages between these two approaches (like with the compile and everything) and he said no! Both should work but your approach makes it clearer for others that there's a css file for the component. :) Great job!!